pkg/scanners/trivy/types.go

```diff
@@ -186,13 +191,13 @@ func (s *ImageScanner) Scan(img unversioned.Image) (ScanStatus, error) {
 		stderr := new(bytes.Buffer)
 
 		cliArgs := s.config.cliArgs(refs[i])
-		cmd := exec.Command(trivyCommandName, cliArgs...)
+		cmd := exec.Command(s.trivyPath, cliArgs...) // nolint:gosec // G204: Subprocess launched with variable
```

Comment:
Can you make the `initScanner` function create a symlink at `/trivy` to the path where trivy was found? That way we don't have to disable a security lint.

Reply:
good point, this might be why we used `/trivy` path instead of relying on `PATH`. However, will this break read only file system pod security?